machine_learning/Stepwise Regression

```diff
@@ -0,0 +1,64 @@
+from sklearn.datasets import load_boston
+import pandas as pd
+import numpy as np
+import statsmodels.api as sm
+
+data = load_boston()
```

Comment:
You should put the test dataset and `X` and `y` variables in an `if __name__ == "__main__"` block at the end of the file:

```python
# End of file
if __name__ == "__main__":
    data = load_boston()
    X = pd.DataFrame(data.data, columns=data.feature_names)
    y = data.target

    result = stepwise_selection(X, y)

    print('resulting features:')
    print(result)